modules/platforms/dotnet/Apache.Ignite.Core.Tests/Client/Cache/CacheTest.cs

```diff
@@ -916,6 +917,151 @@ public void TestCacheNames()
             }
         }
 
+        /// <summary>
+        /// Test cache with expiry policy for Create action.
+        /// </summary>
+        [Test]
+        public void TestCacheWithExpiryPolicyOnCreate()
+        {
+            const int val = 3;
+            var expiryPolicy = new ExpiryPolicy(TimeSpan.FromMilliseconds(100), null, null);
+            var cacheWithExpiryPolicy = GetClientCache<int>().WithExpiryPolicy(expiryPolicy);
+
+            cacheWithExpiryPolicy.Put(val, val);
+
+            // Initially added value is the same.
+            Assert.IsTrue(cacheWithExpiryPolicy.ContainsKey(val));
+
+            // Wait for an expiration.
+            Thread.Sleep(200);
+
+            // Expiry policies should be applied, no cache item exists.
+            Assert.IsFalse(cacheWithExpiryPolicy.ContainsKey(val));
```

Comment:
Here and below - let's add a check for original cache too (without expiry policy).